src/Microsoft.AspNetCore.BuildTools.ApiCheck/ApiListingComparer.cs

```diff
@@ -13,128 +13,108 @@ public class ApiListingComparer
     {
         private readonly ApiListing _newApiListing;
         private readonly ApiListing _oldApiListing;
-        private readonly IList<ApiChangeExclusion> _exclusions;
 
         public ApiListingComparer(
             ApiListing oldApiListing,
-            ApiListing newApiListing,
-            IList<ApiChangeExclusion> exclusions = null)
+            ApiListing newApiListing)
         {
             _oldApiListing = oldApiListing;
             _newApiListing = newApiListing;
-            _exclusions = exclusions ?? new List<ApiChangeExclusion>();
         }
 
-        public ApiComparisonResult GetDifferences()
+        public IList<BreakingChange> GetDifferences()
```

Comment:
The sole task or ApiListingComparer is to find differences between 2 ApiListing, removing known breaking changes should not be part of its tasks.